docs/guides/VTEX-Platform-Overview/security.md

```diff
@@ -0,0 +1,181 @@
+
+## Security incident response
+
+VTEX has a formalized protocol for Security Incident Response, covering the essential stages of the process: Preparation, Incident Identification, Containment, Eradication, Recovery and Post-Incident. This procedure also includes an integrated communication plan that is applied at all stages of the response. Learn more at our [Security Practices](https://vtex.com/us-en/security/security-practices/) document.
+
+### Disaster Recovery Plan
+
+VTEX has a Disaster Recovery plan focused on ensuring operational continuity and the availability of critical resources in emergency situations. This plan is tested at least once a year to ensure its effectiveness. It covers detailed guidelines on how to act in the face of unplanned incidents, characterized as crises, which can arise from natural disasters, cyber attacks or any other disruptive events.
+
+We incorporate crucial recovery steps, including defining our standard RPO (Recovery Point Objective) and RTO (Recovery Time Objective) objectives, as well as carrying out backup tests, ensuring the plan's effectiveness and readiness in real scenarios.
+
+Learn more about our standard RPO and RTO at the [Security FAQ](https://help.vtex.com/en/tutorial/risk-assessment--4K97IpqkpCpDYsZOSxtfkX) – you must have access to an account’s Admin to view this information.
+
+### Business Continuity Plan
+
+With a Business Continuity Plan, we address challenges from emergencies, ensuring a smooth transition and minimizing impacts. This plan establishes a solid foundation for the rapid and effective resumption of normal operations, regardless of the nature of the emergency. Learn more at our [Security Practices](https://vtex.com/us-en/security/security-practices/) document.
+
+## VTEX Shield
+
+[VTEX Shield](https://help.vtex.com/en/tutorial/vtex-shield--2CVk6H9eY2CBtHjtDI7BFh) is an add-on product that offers supplementary, customizable protection layers for stores that prioritize platform resilience and the security standards guaranteed by VTEX's existing security [certifications and practices](https://vtex.com/br-pt/security/security-practices/).
+
+When requesting VTEX Shield, you can choose from the following features:
+
+* [Security Monitor](https://help.vtex.com/en/tutorial/vtex-shield--2CVk6H9eY2CBtHjtDI7BFh#security-monitor)
+* [Penetration tests](https://help.vtex.com/en/tutorial/vtex-shield--2CVk6H9eY2CBtHjtDI7BFh#penetration-tests)
+* [Web Application Firewall (WAF)](https://help.vtex.com/en/tutorial/vtex-shield--2CVk6H9eY2CBtHjtDI7BFh#web-application-firewall-waf)
+
+>ℹ If you are already a VTEX customer and want to adopt VTEX Shield for your business, please contact [Commercial Support](https://help.vtex.com/en/tracks/support-at-vtex--4AXsGdGHqExp9ZkiNq9eMy/3KQWGgkPOwbFTPfBxL7YwZ). Additional fees may apply. If you are not yet a customer but are interested in this solution, please complete our [contact form](https://vtex.com/us-en/contact/).
```

Comment:
Do we need mention in the callout that fees may apply? I'm thinking that including this information here might discourage the merchant from even being interested in learning more about the product.

Reply:
Good point. As a standard for add-ons, we usually do inform that to avoid false expectations that they'll reach out to support and get access for free, so I think it's best to leave it for now 🤔 But with more add-on products being created, we have an opportunity to rethink that in the future!